packages/moonstone/Panels/Header.module.less

```diff
@@ -86,19 +87,24 @@
 		.inputHighlight {
 			max-width: 100%;
 		}
-
 	}
 
 	// Standard Header
 	&.standard {
+		height: @moon-header-standard-height;
+
 		.titlesCell {
 			margin-bottom: 9px;
 		}
+
+		&.minimized {
+			height: (@moon-header-standard-height - 89px);  // Must subtract the height of the header element. Considering changing the `em` line-height to a px value to make this more straightforward.
```

Comment:
Is `89px` something special?  Why is this not a multiple of 3?

Reply:
It's vestigial, from the previous commit. I'll update it to 90. 89 likely came from the browser's measurement of the node, since the node's size is based on the font size and `em`.